gui/src/components/BalancesList.tsx

```diff
@@ -32,17 +32,16 @@ export function BalancesList() {
 }
 
 function BalanceETH() {
-  const address = useAccount();
   const { currentNetwork } = useCurrentNetwork();
-  const { data: balance } = useBalance({
-    address,
-    enabled: !!address,
-    chainId: currentNetwork?.chain_id,
-  });
-
-  if (!balance) return null;
+  const { balance } = useNativeBalance();
 
-  return <BalanceItem balance={balance.value} decimals={18} symbol="ETH" />;
+  return (
+    <BalanceItem
+      balance={balance}
+      decimals={currentNetwork?.decimals || 18}
+      symbol={currentNetwork?.currency || "ETH"}
```

Comment:
I think instead of this we can just do `if (!currentNetwork) return null;` right?